packages/api/src/webhooks/cannon.test.ts

```diff
@@ -219,29 +228,34 @@ describe("webhook cannon", () => {
       expect(res.status).toBe(201);
       res = await client.post("/webhook", {
         ...mockWebhook,
+        url: mockWebhook.url + "2",
         name: "test-2",
       });
       expect(res.status).toBe(201);
 
-      const sem = semaphore();
-      let callCount = 0;
+      const semaphs = [semaphore(), semaphore()];
```

Comment:
`sems` not good enough for you? 😁

Reply:
It actually sounded a bit cacophonic to me, but no biggie. Will change to `sems` 😁